### DISCUSSION TITLE
use xdp example, but return operation not supported

### DISCUSSION THREAD

**Discussion by yuri-lcy** [cilium/ebpf#1317] [Q&A] [answered]
![image](https://github.com/cilium/ebpf/assets/114134743/c05810e3-1561-4397-b951-b95b1cc2eff2)
kernel is  5.10.134

**Comment by lmb** [comment, marked as answer]
Hi, this forum is meant for questions and problems with the library. Your problem is likely with your hardware / driver.

Posting a screenshot instead of copy and pasting the text also means that other people can't benefit from any help you receive here, since they won't be able to search for the problem. Hence I'm closing this issue.
